ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -72,10 +72,12 @@
   private final List<NetworkReceive> completedReceives;
   private final List<String> disconnected;
   private final List<String> connected;
+  private final Map<String, TransmissionPendingHandshake> pendingSslTrans;
```

Comment:
could be consistent with calling these pending things `Transmission` or `Connection`. I would prefer using `ConnectionPendingHandshake`.
Once again, I think shortening `Transmission` to `Trans` is very awkward.